code/global.dm

```diff
@@ -376,6 +376,28 @@ var/global
 	"Yesteryear",
 	"Zeyada")
 
+	list/selectable_ringtones = list("Two-Beep" = new/datum/ringtone,
```

Comment:
filteredtypesof?

Reply:
Did that, looks like it works nicely, much better than manually defining them all. Now, availability is defined by a var in the ringtone itself.